decentraland/unity-renderer
# refactor: gif processing for better performance and multithreading

## What does this PR change?

UniGif was deleted, now we use Unity Gif Decoder from https://github.com/3DI70R/Unity-GifDecoder

Fixes #1842 

## How to test the changes?

Desktop Only:
Run through the world and pass-by scenes with lots of gifs, there shouldn't be performance spikes.

A good example is -14, -12 ( it has 4 gifs and a broken URL)

WebGL:
Should work like before

Unity:
There's a test scene named UniGifStressTest, you can try it by toggling the multithreaded bool to see the difference in performance using the profiler.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
